Makefile

```diff
@@ -37,3 +38,6 @@ kingpin.zip:
 	cd zip; ln -sf kingpin/bin/deploy.py ./__main__.py
 	cd zip; zip -9mrv ../kingpin.zip .
 	rm -rf zip
+
+docs:
+	make -C docs html
```

Comment:
Always use $(MAKE) here and on line 21 -- it'll propagate all the flags and settings along with calling the function